### PR TITLE
Links in Docstrings

### PR DESCRIPTION
**Pull request by ksunden**

### CODE REVIEW

File: WrightTools/kit.py [hunk 1]
```diff
@@ -637,7 +644,7 @@ def read_headers(filepath):
 
 
 def write_h5(filepath, dictionary):
-    """ Save a python dictionary into an `HDF5 <https://www.hdfgroup.org/HDF5/doc/H5.intro.html>`_
+    """ Save a python dictionary into an `HDF5`_
```
Comment (untzag) [inline WrightTools/kit.py]: does this link inherit from the docstring earlier in the file?

Comment (ksunden) [inline WrightTools/kit.py]: It does, and it throws errors if you try to put the same tag in but redefine it...

I don't like not having the link in the docstring itself, but I'm not sure how to resolve that at this time

**Comment by ksunden** [comment]
I found out rST does allow hyperlinks to be split on multiple lines (and ignores whitespace) This will remove the majority of remaining line length errors

File: WrightTools/kit.py (hunk 1 above)
Comment (ksunden) [inline WrightTools/kit.py]: One thing we could do is use anonymous hyperlink in this case, i.e. `__` and then have 
```
.. __: hdf5-write_
.. _hdf5-write: https://hdf5.org/path/to/index.html
```

Comment (ksunden) [inline WrightTools/kit.py]: This would allow us to have unique names for the links themselves in underlying rST

Comment (untzag) [inline WrightTools/kit.py]: in this case you're the expert @ksunden---whatever you think is the best strategy

**Comment by untzag** [comment]
@ksunden are you working in this branch or shall I merge now?

**Comment by ksunden** [comment]
I plan on cleaning up a few more bits, please leave this open for the time being

**Pull request merged**